pkg/agent/openflow/client.go

```diff
@@ -40,6 +40,9 @@ type Client interface {
 	// InstallGatewayFlows sets up flows related to an OVS gateway port, the gateway must exist.
 	InstallGatewayFlows(gatewayAddr net.IP, gatewayMAC net.HardwareAddr, gatewayOFPort uint32) error
 
+	// InstallHostNetworkFlows installs Openflow entries for uplink/bridge to support host networking.
+	InstallHostNetworkFlows(uplinkPort uint32, bridgeLocalPort uint32) error
```

Comment:
I feel we need a better name closer to the function of this func. How about InstallBridgePortFlows, or InstallBridgeUplinkFlows?

Reply:
Thanks Jianjun, I think `InstallBridgeUplinkFlows` is better, rename the method in update.